index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Dyary's Portfolio</title>
+        <link rel="stylesheet" href="styles.css">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+    <body>
+        <div class="container">
+            <nav>
+                <ul>
+                    <li id="my-logo-nav-item">My Logo</li>
+                    <li class="left-nav-items">Portfolio</li>
+                    <li class="left-nav-items">about</li>
+                    <li class="left-nav-items">contact</li>
+                    <li id="hamburger-nav-item">
+                        <img src="images/hamburger-icon.png" alt="hamburger
+                            icon">
+                    </li>
+                </ul>
+            </nav>
```

Comment:
- [ ] Please ensure to wrap this code(the header code) inside the `<header>` tag. For instance, you could replace the `<nav>` tag with the `<header>` tag. This is in order to achieve high levels of semantics in your code.